shared/gh/js/controllers/gh.api.util.js

```diff
@@ -139,25 +137,104 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
     };
 
     /**
-     * Get the number of weeks in a term
+     * Return the current academic week number if the current date is within a term
      *
-     * @param  {Object}    term    The term to get the number of weeks for
-     * @return {Number}            The number of weeks in the term
+     *  * To calculate the academic week number we first need to calculate
+     *  * the difference between the current calendar date and the start of the term.
+     *  *
+     *  * Note that a term always start on a Tuesday and academic weeks start on Thursdays!
+     *  *
+     *  * E.g: calendar date: Wed 2015-02-11T13:54:35.000Z => 1423662834000 (week 1)
+     *  *      start of term: Thu 2015-01-13T00:00:00.000Z => 1421107200000 (week 5)
+     *  *
+     *  * This means that the first academic week start on Tuesday, 13/01/2015 and ends on Wednesday, 14/01/2015.
+     *  * The second week will start on Thursday, 15/01/2015 and end on Wednesday 21/01/2015.
+     *  *
+     *  * After we calculated the difference between the current date and the start of the term,
+     *  * we need the number of weeks:
+     *  *
+     *  *   weeks = (calendar date - term start) / (week in milliseconds)
+     *  *   weeks = (1423662834000 - 1421107200000) / (1000 * 60 * 60 * 24 * 7)
+     *  *   weeks = 4.226...
+     *  *
+     *  * We need to subtract an offset from our result, since there is a difference of
+     *  * 2 days between the start of the term (Tue) and the start of the academic weeks (Thu).
+     *  *
+     *  *   offset = ((1 / 7) * 2)
```

Comment:
We shouldn't assume that the terms will always start on a Tuesday. This offset should be calculated by taking the first day of the week (always a Thursday) and calculating the offset by getting the difference between it and the actual start of the week.